src/pages/manager-review-dashboard/useFilters.ts

```diff
@@ -33,14 +33,16 @@ export const useFilters = () => {
   };
 
   const filterStatus = (row: Row) => {
+    if (filters.status === 'all') {
+      return true;
+    }
     if (filters.status === 'todo') {
       return row.behavioralCompetencies === 0 && row.achievements === 0;
     }
     if (filters.status === 'doing') {
-      return !(
-        (row.behavioralCompetencies === 100 && row.achievements === 100) ||
-        (row.behavioralCompetencies === 0 && row.achievements === 0)
-      );
+      const done = row.behavioralCompetencies === 100 && row.achievements === 100;
+      const todo = row.behavioralCompetencies === 0 && row.achievements === 0;
```

Comment:
Maybe we could use the `done` and `todo` variables defined here in 
`filters.status === 'todo'`
and 
filters.status === 'done'